src/devices/tuya.ts

```diff
@@ -5503,6 +5503,31 @@ const definitions: Definition[] = [
             e.binary('garage_door_contact', ea.STATE, false, true)
                 .withDescription('Indicates if the garage door contact is closed (= true) or open (= false)')],
     },
+    {
+        fingerprint: [{modelID: 'TS0603', manufacturerName: '_TZE608_c75zqghm'}],
+        zigbeeModel: ['TS0603'],
+        model: 'TS0603',
+        vendor: '_TZE608_c75zqghm',
+        extend: [],
+        meta: {
+            tuyaDatapoints: [
+                [1, null, null],
+                [3, 'garage_door_contact', tuya.valueConverter.trueFalseInvert],
+                [12, null, null],
+            ],
+        },
+        description: 'Garage door opener',
+        configure: async (device, coordinatorEndpoint) => {
+            await tuya.configureMagicPacket(device, coordinatorEndpoint);
+            const endpoint = device.getEndpoint(1);
+            await reporting.bind(endpoint, coordinatorEndpoint, ['genBasic']);
+        },
+        fromZigbee: [legacy.fromZigbee.matsee_garage_door_opener, fz.ignore_basic_report, tuya.fz.datapoints],
```

Comment:
```suggestion
        fromZigbee: [tuya.fz.datapoints],
```

Can you use this and add everything to `meta. tuyaDatapoints` instead?

(same for `toZigbee`)